### ISSUE TITLE
ARInside Helix 21.3

### ISSUE THREAD

**Issue by croth308** [gabeluci/ARInside#3]
We tried the latest version of ARInside (3.1.4) against a Helix 21.3 Remedy environment using the slow load switch, but it never completes. It just hangs.

**Comment by gabeluci** [comment]
How long have you let it run? In our environment, it takes 2.5 hours for it to complete.

**Comment by croth308** [comment]
Several hours - let me try again. Does it work for you?


Charles Roth
Contractor - NetImpact Strategies
Governance Services Division/Governance Application Management Branch
United States Department of Agriculture
Mobile: (864) 616-2975

**Comment by gabeluci** [comment]
It does work for us. Can you put `-v >arinside.log 2>&1` at the end of the command line and then attach arinside.log file here once you're convinced it's hanging? (I believe you can open arinside.log while it's running to see the progress) I'd like to see at what point it's hanging.

**Comment by croth308** [comment]
It finished, I let it run all night. Its just very slow.

Thanks

Charles Roth
Contractor - NetImpact Strategies
Governance Services Division/Governance Application Management Branch
United States Department of Agriculture
Mobile: (864) 616-2975

**Comment by gabeluci** [comment]
Ok, glad to hear it works at least.
